src/test/java/space/obminyashka/items_exchange/controller/AuthorizationControllerTest.java

```diff
@@ -30,25 +37,34 @@ class AuthorizationControllerTest {
     private UserService userService;
     @Mock
     private MailService mailService;
+    @Mock
+    private EmailConfirmationTokenRepository confirmationTokenRepository;
     @InjectMocks
     private AuthController authController;
+    @Captor
+    private ArgumentCaptor<EmailConfirmationToken> captor;
     private final UserRegistrationDto dto = new UserRegistrationDto("user", "user@mail.ua", "pass", "pass");
+    private User userFromDto = new User();
```

Comment:
Why the blank user is created directly here instead of doing that via BeforeEach method?